mock_data/main.py

```diff
@@ -0,0 +1,226 @@
+import json
+from uuid import uuid4
+import os
+import argparse
+import re
+import datetime
+import random
+
+from progress.bar import IncrementalBar as Bar 
+from openai import OpenAI
+import requests
+
+# STEP 0: Initialize constants + utility/helper functions
+parser = argparse.ArgumentParser(prog="Mock data generator",
+                                 description="Generates mock club, category, and tag data for testing/development.")
+parser.add_argument("output_file", help="The SQL file to output the data to.")
+parser.add_argument("-a", "--api-key", default=os.environ.get("OPENAI_API_KEY", ""), help="The OpenAI API key to use for generating descriptions and tags for clubs. (default: key in OPENAI_API_KEY or '' if that doesn't exist.)")
+parser.add_argument("-t", "--top-n", default=1024, help="The max # of clubs to retrieve from NUEngage. (default: 1024)")
+parser.add_argument("-p", "--parent_uuid", default="00000000-0000-0000-0000-000000000000", help="The UUID of the parent club for the generated clubs. (default: 00000000-0000-0000-0000-000000000000)")
+args = parser.parse_args()
+
+openai_client = OpenAI(api_key=args.api_key)
+OUTPUT_FILE = args.output_file
+TOP_N = args.top_n
+PARENT_UUID = args.parent_uuid
+
+MOCK_CATEGORIES_AND_TAGS = {
+    "PreProfessional": [
+        "Premed",
+        "Prelaw",
+    ],
+    "CulturalAndIdentity": [
+        "Judaism",
+        "Christianity",
+        "Hinduism",
+        "Islam",
+        "LatinAmerica",
+        "AfricanAmerican",
+        "AsianAmerican",
+        "LGBTQ",
+    ],
+    "ArtsAndCreativity": [
+        "PerformingArts",
+        "VisualArts",
+        "CreativeWriting",
+        "Music",
+    ],
+    "SportsAndRecreation": [
+        "Soccer",
+        "Hiking",
+        "Climbing",
+        "Lacrosse",
+    ],
+    "ScienceAndTechnology": [
+        "Mathematics",
+        "Physics",
+        "Biology",
+        "Chemistry",
+        "EnvironmentalScience",
+        "Geology",
+        "Neuroscience",
+        "Psychology",
+        "SoftwareEngineering",
+        "ArtificialIntelligence",
+        "DataScience",
+        "MechanicalEngineering",
+        "ElectricalEngineering",
+        "IndustrialEngineering",
+    ],
+    "CommunityServiceAndAdvocacy": [
+        "Volunteerism",
+        "EnvironmentalAdvocacy",
+        "HumanRights",
+        "CommunityOutreach",
+    ],
+    "MediaAndCommunication": [
+        "Journalism",
+        "Broadcasting",
+        "Film",
+        "PublicRelations",
+    ]
+}
+
+TAGS_STRING = "" # for tags_prompt
+for key, value in MOCK_CATEGORIES_AND_TAGS.items():
+    for tag in value:
+        TAGS_STRING += f"\"{tag}\", "
+TAGS_STRING = TAGS_STRING[:-2] # remove the last ", "
+
+def description_prompt(name, preview):
```

Comment:
i'm a big type hint head, please forgive these nits

```suggestion
def description_prompt(name: str, preview: str) -> str:
```